tests/agent/service/notificationsSend.test.ts

```diff
@@ -26,7 +26,7 @@ import * as utilsPB from '@/proto/js/polykey/v1/utils/utils_pb';
 import * as notificationsPB from '@/proto/js/polykey/v1/notifications/notifications_pb';
 import * as nodesUtils from '@/nodes/utils';
 import * as notificationsUtils from '@/notifications/utils';
-import { expectRemoteError } from '../../utils';
+import { expectRemoteError } from '../../utils/utils';
```

Comment:
Just do `import * as testUtils from '../../utils';`.

Reply:
Same with the other places.

Reply:
I thought we were moving away from index imports? Or is it fine for these utils?

Reply:
Fine for utilities here.

Reply:
Fixed now for all places